src/components/styles/Calculator.module.css

```diff
@@ -0,0 +1,11 @@
+.calculatorDiv {
+  display: flex;
+  width: 100%;
+  height: 80vh;
+}
+
+.calculatorDiv > h2 {
+  font-size: 30px;
+  text-align: center;
+  width: 40%;
+}
```

Comment:
- Kindly fix some issues below:
  - [ ] Kindly match the basic structure with the wireframe. The `Math Magicians` logo should be aligned with the `Let's do some math!` and other content of the page.
  - [ ] Kindly match the basic structure with the wireframe. The `Let's do some` text should be aligned with the `Calculator interface` horizontally.
  
  ![image](https://user-images.githubusercontent.com/60083437/228642487-dbc11159-9fec-4eb9-9ad1-cd682b2be99f.png)